server/src/main/java/com/cloud/hypervisor/HypervisorGuruBase.java

```diff
@@ -97,6 +98,24 @@
     public static ConfigKey<Boolean> VmMinCpuSpeedEqualsCpuSpeedDividedByCpuOverprovisioningFactor = new ConfigKey<Boolean>("Advanced", Boolean.class, "vm.min.cpu.speed.equals.cpu.speed.divided.by.cpu.overprovisioning.factor", "true",
             "If we set this to 'true', a minimum CPU speed (cpu speed/ cpu.overprovisioning.factor) will be set on the VM, independent of using a scalable service offering or not.", true, ConfigKey.Scope.Cluster);
 
+    private Map<NetworkOffering.Detail, String> getNicDetails(Network network) {
+        Map<NetworkOffering.Detail, String> details = null;
+        if (network != null) {
```

Comment:
We could invert this if to reduce indentation and maybe add a log.

Reply:
@GutoVeronezi done